app/views/cloud_object_store_container/new.html.haml

```diff
@@ -0,0 +1,62 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "cloudObjectStoreContainerFormController"}
```

Comment:
Just checking: did you notice any flicker when you open this page? First, browser default controls are displayed for a short period, then proper controls used by miq. If you did, check `ng-cloak`.